python/py-setuptools-rust/Portfile

```diff
@@ -31,6 +31,7 @@ if {${name} ne ${subport}} {
     depends_run-append \
                     port:cargo \
                     port:py${python.version}-semantic_version \
+                    port:py${python.version}-toml \
```

Comment:
this shouldn't be added here since it is not needed at runtime as far as I can tell (and if you would want to add it also as runtime dependency, it should be listed once under `depends_lib` and then remove from `depends_build`)

Reply:
Moved to depends_lib